docker-storage-setup.sh

```diff
@@ -633,15 +641,8 @@ is_valid_storage_driver() {
 get_existing_storage_driver() {
   local options driver
 
-  if [ ! -f "$DOCKER_STORAGE" ];then
-    return 0
-  fi
-
-  if ! options=$(get_docker_storage_options); then
-    return 1
-  fi
+  options=$CURRENT_STORAGE_OPTIONS
```

Comment:
Hmm..., not sure what you did not like. Did you not like assigning CURRENT_STORAGE_OPTIONS to local variable or you did not like querying storage options and saving it in a global variable.